package.json

```diff
@@ -5,7 +5,9 @@
   "scripts": {
     "clean": "rm -rf ./dist",
     "prebuild": "npm run clean",
+    "dev:ovp:massive": "webpack-dev-server --config webpack.config.massive.ovp.js",
```

Comment:
Rename `massive` into `no-ui` everywhere please.

Reply:
It is not only UI that is excluded lets call it `minimal`

Reply:
And squash into one commit please